src/Message/InquireRequest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Omnipay\Cardconnect\Message;
+
+class InquireRequest extends AbstractRequest
+{
+    public function getData()
+    {
+        $this->validate('transactionReference');
+    	$data = array(
+    		'merchid'   => $this->getMerchantId(),
+    		'retref'   => $this->getTransactionReference()
+    	);
+        return $data;
+    }
+
+    public function sendData($data)
+    {
+        $response = $this->httpClient->request('GET', $this->getEndpoint($data), $this->getHeaders());
+
+        if ($response->getStatusCode() != 200) {
+            throw new \Exception($response->getReasonPhrase());
+        }
+
+        $json = json_decode($response->getBody()->getContents(), true);
+
+        $this->response = new Response($this, [
+            'respcode' => $json['respcode'] ?? null,
+            'voidable' => isset($json['voidable']) && $json['voidable'] === 'Y',
+            'refundable' => isset($json['refundable']) && $json['refundable'] === 'Y',
+        ]);
+
+        return $this->response;
+    }
+
+    public function getEndpoint($data)
+    {
+        return $this->getEndpointBase() . "/inquire/{$data['retref']}/{$data['merchid']}";
+    }
+}
```

Comment:
Please add a newline to the end of this file.
![image](https://user-images.githubusercontent.com/63778085/180047768-d29edeca-a485-4e27-b362-9a725ebe66f2.png)

Reply:
lol